solarforecastarbiter/plotting/utils.py

```diff
@@ -77,3 +77,23 @@ def line_or_step(interval_label):
             '"event", or "ending"')
 
     return plot_method, plot_kwargs, hover_kwargs
+
+
+def line_or_step_plotly(interval_label):
+    """
+    For a given interval_label, determine any kwargs for the plot.
+    """
+    if 'instant' in interval_label:
+        plot_kwargs = dict()
+    elif interval_label == 'beginning':
+        plot_kwargs = dict(line_shape='hv')
+    elif interval_label == 'ending':
+        plot_kwargs = dict(line_shape='vh')
+    elif interval_label == 'event':
+        plot_kwargs = dict(line_shape='vh')
```

Comment:
To me 'hv' makes more sense for events. Basically, the idea is that once an event switches, you would expect it to continue at 1/0 until the next switch and the line after the point kinda signals that. :shrug:

Reply:
True. I just copied the bokeh setting, but will change both.

<img width="875" alt="Screen Shot 2020-04-24 at 3 14 01 PM" src="https://user-images.githubusercontent.com/4383303/80261077-4a3fa200-863e-11ea-8586-1a62c30f86bd.png">